src/scicat_offline_ingestor.py

```diff
@@ -149,9 +149,9 @@ def _open_h5file(
         # that can result non-stopped generator error.
         # It happens when it doesn't fail opening the file but fails after.
         opened_file = h5py.File(file_path, 'r')
-    except BlockingIOError as e:
+    except Exception as e:
```

Comment:
```suggestion
    except (OSError, BlockingIOError) as e:
```

This should be enough I think... I couldn't find when exaclty it throws `OSError` and `BlockingIOError` but it seems like both of them are related to file locking. And `OSError` can happen more often...